Druid/DRBytecodeGenerator.class.st

```diff
@@ -197,9 +197,16 @@ DRBytecodeGenerator >> visitLoadArgument: aDRLoadArgument [
 { #category : 'ir-to-target' }
 DRBytecodeGenerator >> visitLoadInstVar: aDRLoadInstVar [ 
 	
-	aDRLoadInstVar operand1 acceptVisitor: self.
-	builder pushLiteral: aDRLoadInstVar index.
-	builder send: #instVarAt:
+	aDRLoadInstVar operand1 acceptVisitor: self. 
```

Comment:
This should not be inside the if?

This visit push the object to access, but this is only necessary for the _message send case_ `send: #instVarAt:`.
The `pushInstVar` bytecode is always on the receiver (that's the original problem), so we don't need to push it before (it's implicit).